1.9/usage/service-discovery/load-balancing-vips/index.md

```diff
@@ -4,7 +4,7 @@ nav_title: Load Balancing and VIPs
 menu_order: 50
 ---
 
-DC/OS comes with an east-west load balancer that's meant to be used to enable multi-tier microservices architectures. It acts as a TCP Layer 4 load balancer, and it's tightly integrated with the kernel. The internal Layer 4 load balancer component (`dcos-minuteman.service`) is also known as [Minuteman](https://github.com/dcos/minuteman). 
+DC/OS comes with an east-west load balancer that's meant to be used to enable multi-tier microservices architectures. It acts as a TCP Layer 4 load balancer, and it's tightly integrated with the kernel. The internal Layer 4 load balancer component (which is part of the `dcos-navstar.service`) is also known as [Minuteman](https://github.com/dcos/minuteman).
```

Comment:
@karlkfi anchors don't work [because your page is in HTML](https://jira.mesosphere.com/browse/DCOS_SITE-131). You'll need to add manual anchors for this to work.